### PR TITLE
LibWeb: Some FIXME hunting :^)

### PR DESCRIPTION
**Pull request by jamierocks**
This pull request resolves 6 FIXMEs in the project :^)

### CODE REVIEW

File: Userland/Libraries/LibWeb/HTML/EventNames.h [hunk 1]
```diff
@@ -104,6 +104,7 @@ namespace Web::HTML::EventNames {
     __ENUMERATE_HTML_EVENT(stalled)                  \
     __ENUMERATE_HTML_EVENT(statechange)              \
     __ENUMERATE_HTML_EVENT(storage)                  \
+    __ENUMERATE_HTML_EVENT(success)                  \
     __ENUMERATE_HTML_EVENT(submit)                   \
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/HTML/EventNames.h:108]: Nit: success comes after submit

File: Userland/Libraries/LibWeb/CSS/MediaQuery.cpp [hunk 2]
```diff
@@ -392,60 +392,6 @@ String serialize_a_media_query_list(Vector<NonnullRefPtr<MediaQuery>> const& med
     return MUST(String::join(", "sv, media_queries));
 }
 
-bool is_media_feature_name(StringView name)
```
Comment (AtkinsSJ) [inline Userland/Libraries/LibWeb/CSS/MediaQuery.cpp:395]: :laughing: Nice catch! Can't believe this hung around for 2 years.

**Review by AtkinsSJ — CHANGES_REQUESTED** [review]

File: Userland/Libraries/LibWeb/CSS/MediaQuery.cpp (hunk 2 above)
Comment (jamierocks) [reply Userland/Libraries/LibWeb/CSS/MediaQuery.cpp:395]: I must admit - I realised it was no longer used, after resolving the FIXME. :cry:

File: Userland/Libraries/LibWeb/HTML/EventNames.h (hunk 1 above)
Comment (jamierocks) [reply Userland/Libraries/LibWeb/HTML/EventNames.h]: Resolved :)

File: Userland/Libraries/LibWeb/CSS/MediaQuery.cpp (hunk 2 above)
Comment (AtkinsSJ) [reply Userland/Libraries/LibWeb/CSS/MediaQuery.cpp:395]: Sorry! :sweat_smile:

**Review by AtkinsSJ — APPROVED** [review]

**Pull request merged**